### PR TITLE
Fix #1959: Optimize admin_auth_activity.xml

### LINKED ISSUE #1959: Optimise code in admin_auth_activity.xml
This issue is tracking the optimization work of `admin_auth_activity.xml`

1. Make sure that the layout uses `ConstraintLayout` and there is minimal nesting of views. If this view already uses `ConstraintLayout` then it's fine.

2. Make sure that all TextViews use the style from `styles.xml` file like `Heading1`, `TextFieldLabel`, `Body`, etc. whichever suits best.

While making the above changes the UI should not get affected and the end result should still be the same as before (or an optimized version of UI).

If there are multiple files with the same name, update them all.

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

Fixes #1959 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

## Screenshots after changes
Tablet portrait | Tablet Landscape
--- | ---
Screenshot_1607024674 | Screenshot_1607024677 |

Mobile portrait | Mobile Landscape
--- | ---
 Screenshot_1607024958 | Screenshot_1607024968

### CODE REVIEW

**Comment by FareesHussain** [comment]
Checks show theres a error in testWorker_logEvent_withoutNetwork_enqueueRequest_verifySuccess
image
but it passes when I run it in local
image

**Comment by rt4914** [comment]
> Checks show theres a error in testWorker_logEvent_withoutNetwork_enqueueRequest_verifySuccess
> image
> but it passes when I run it in local
> image

Restarted the CI job. It should pass now.

File: app/src/main/res/layout-land/admin_auth_activity.xml [hunk 1]
```diff
@@ -61,30 +61,26 @@
...
 
           <TextView
             android:id="@+id/admin_auth_sub_text"
+            style="@style/Subtitle1"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="12dp"
-            android:fontFamily="sans-serif"
             android:text="@string/admin_auth_sub"
-            android:textColor="@color/oppiaAdminAuthSecondaryText"
```
Comment (rt4914) [inline app/src/main/res/layout-land/admin_auth_activity.xml:86]: Please add this color back. You can notice that color this textview is different in screenshots.

File: app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml [hunk 2]
```diff
@@ -63,28 +63,24 @@
...
 
           <TextView
             android:id="@+id/admin_auth_sub_text"
+            style="@style/Subtitle1"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="12dp"
-            android:fontFamily="sans-serif"
             android:text="@string/admin_auth_sub"
-            android:textColor="@color/oppiaAdminAuthSecondaryText"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml:86]: Ditto

File: app/src/main/res/layout-sw600dp-port/admin_auth_activity.xml [hunk 3]
```diff
@@ -56,27 +56,23 @@
...
 
         <TextView
           android:id="@+id/admin_auth_sub_text"
+          style="@style/Subtitle1"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
           android:layout_marginTop="12dp"
-          android:fontFamily="sans-serif"
           android:text="@string/admin_auth_sub"
-          android:textColor="@color/oppiaAdminAuthSecondaryText"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/admin_auth_activity.xml:78]: Ditto

File: app/src/main/res/layout/admin_auth_activity.xml [hunk 4]
```diff
@@ -54,31 +54,27 @@
...
           android:id="@+id/admin_auth_sub_text"
+          style="@style/Subtitle1"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
           android:layout_marginStart="28dp"
           android:layout_marginTop="12dp"
           android:layout_marginEnd="28dp"
-          android:fontFamily="sans-serif"
           android:text="@string/admin_auth_sub"
-          android:textColor="@color/oppiaAdminAuthSecondaryText"
```
Comment (rt4914) [inline app/src/main/res/layout/admin_auth_activity.xml:80]: Ditto

**Review by rt4914 — CHANGES_REQUESTED** [review]
@FareesHussain Suggested changes.

File: app/src/main/res/layout-land/admin_auth_activity.xml (hunk 1 above)
Comment (FareesHussain) [reply app/src/main/res/layout-land/admin_auth_activity.xml:86]: Yes there's a slight change now i've added it

File: app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml (hunk 2 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/admin_auth_activity.xml:86]: Done

File: app/src/main/res/layout-sw600dp-port/admin_auth_activity.xml (hunk 3 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-port/admin_auth_activity.xml:78]: Done

File: app/src/main/res/layout/admin_auth_activity.xml (hunk 4 above)
Comment (FareesHussain) [reply app/src/main/res/layout/admin_auth_activity.xml:80]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**